pkg/secrets/clients/k8s/k8s_secrets_client.go

```diff
@@ -32,15 +28,21 @@ func PatchK8sSecret(namespace string, secretName string, stringDataEntriesMap ma
 	// get K8s client object
 	kubeClient, _ := configK8sClient()
 
-	stringDataEntry, err := generateStringDataEntry(stringDataEntriesMap)
+	log.Info(messages.CSPFK006I, secretName, namespace)
+	k8sSecret, err := kubeClient.CoreV1().Secrets(namespace).Get(secretName, metav1.GetOptions{})
```

Comment:
Why do we need to retrieve the k8s secret here? It is retrieved in `RetrieveK8sSecret` and this method gets its `stringDataEntriesMap`. Please follow the logic [here](https://github.com/cyberark/secrets-provider-for-k8s/blob/master/pkg/secrets/k8s_secrets_storage/provide_conjur_secrets.go#L52) to see how to change this method's logic correctly.

Reply:
done